citest/json_contract/path.py

```diff
@@ -18,6 +18,10 @@
 import collections
 from ..base import JsonSnapshotable
 
+# Specifies the path-delimiter used when specifying paths
+PATH_SEP = '/'
```

Comment:
The use of '/' is coincidence. It is not tied to the OS and I'd like to change it to '.' but '/' is hardcoded into all the existing stuff, so this lets me two-step the conversion over time.